src/view/form-create-view.js

```diff
@@ -1,165 +1,131 @@
 import AbstractView from '../framework/view/abstract-view.js';
+import {DATE_FORMAT, EVENT_POINTS_TYPE} from '../const.js';
+import {humanizeEventDate, createUpperCase} from '../utils.js';
+
+function createTypeTemplate(type) {
+  return (
+    `<div class="event__type-item">
+      <input id="event-type-${type}-1" class="event__type-input  visually-hidden" type="radio" name="event-type" value="${type}">
+      <label class="event__type-label  event__type-label--${type}" for="event-type-${type}-1">${createUpperCase(type)}</label>
+    </div>`
+  );
+}
+
+function createOfferTemplate(offers) {
+  const {id, title, price} = offers;
```

Comment:
```suggestion
function createOfferTemplate(offer) {
  const {id, title, price} = offer;
```
s из названия убери, сюда приходит один объект, и в form-edit тогда тоже поправь